torchgeo/models/fcsiam.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         encoder_name: str = 'resnet34',
         encoder_depth: int = 5,
         encoder_weights: str | None = 'imagenet',
-        decoder_use_batchnorm: bool = True,
+        decoder_use_batchnorm: bool | str | dict[str, Any] = 'batchnorm',
```

Comment:
We should deprecate this parameter and rename it to decoder_use_norm, but in a separate PR so this can still make it into 0.7.1.